### ISSUE TITLE
Black screen on Safari Desktop 12.4 and 13.1

### ISSUE THREAD

**Issue by csalmeida** [rafrex/fscreen#21]
Hey, thanks for putting this package together. 

I've experienced an issue Safari Desktop `12.4` and `13.1` where I get a black screen when applying the fullscreen on a `div` element. The element has a `video` and other elements nested.

Pressing `Esc` will exit fullscreen and return the element to normal.

Any thoughts on why this might be happening? The [supported browsers list](https://caniuse.com/#feat=fullscreen) suggests that these would supported at least partially, maybe there's something I'm missing?

The same element worked with no issues on Firefox 76 and Chrome 81.

**Comment by rafrex** [comment]
I don’t know why this would happen. It seems like it is an issue with Safari and not with this package. That is, does it happen if using the full screen api without fscreen? If it does then I would file an issue with Safari.

**Comment by csalmeida** [comment]
Thanks for getting back to me. Same here but it seems that it does have to do with the Fullscreen API, or how Safari handles it at least. 

I have tried to use the [Fullscreen API](https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API) without fscreen and it seems that the method is not present in this version (`13.1`) of Safari:

```
TypeError: playerWrapper.requestFullscreen is not a function. 
(In 'playerWrapper.requestFullscreen()', 'playerWrapper.requestFullscreen' is undefined)
```

I can see from the fscreen source code that for Safari `webkitRequestFullscreen` is used instead and when I tried that method and when the element has gone full screen I got the same black screen.

https://github.com/rafrex/fscreen/blob/47a4aa3e525a9a0b4f8bf5e686d3d3aac8ecdf53/src/index.js#L14

Closing as it's not an issue of fscreen.
